cmd/scollector/service_windows.go

```diff
@@ -71,7 +73,9 @@ func installService(name, desc string) error {
 		return err
 	}
 	defer s.Close()
-	err = eventlog.InstallAsEventCreate(name, eventlog.Error|eventlog.Warning|eventlog.Info)
+	// Workaround for bug in eventlog.InstallAsEventCreate
```

Comment:
What exactly is the bug? Comparing this to the implementation of InstallAsEventCreate ([here](https://code.google.com/p/winsvc/source/browse/eventlog/install.go#61)), it is identical to your change except that this one calls ExpandEnv and the other one doesn't. Can you describe the bug more?